python/llm/example/CPU/Speculative-Decoding/baichuan2/README.md

```diff
@@ -60,3 +60,28 @@ Tokens generated 128
 E2E Generation time x.xxxxs
 First token latency x.xxxxs
 ```
+
+### 4. Accelerate with BIGDL_OPT_IPEX
+
+To accelerate speculative decoding on CPU, you can install our validated version of [IPEX](https://github.com/intel/intel-extension-for-pytorch/tree/0c63936d7a6740679987920367ae2e0cdb375b2e) by following scripts: (Other versions of IPEX may have some conflicts and can not accelerate speculative decoding correctly.)
+
+```bash
```

Comment:
Better to separate the two operations into two bash cells, and move the notes to main body text instead of annotations.